app/Actions/Diagnostics/Pipes/Checks/AppUrlMatchCheck.php

```diff
@@ -50,7 +61,7 @@ public function handle(array &$data, \Closure $next): array
 			$data[] = 'Error: LYCHEE_UPLOADS_URL is set and empty. This will prevent images to be displayed. Remove the line from your .env';
```

Comment:
```suggestion
			$data[] = 'Error: LYCHEE_UPLOADS_URL is set and empty. This will prevent images from being displayed. Remove the line from your .env';
```